<launch>

  <node name="vicon_client" pkg="vicon_udp" type="receive.py" ns="udp" >
    <param name="object_name" value="QAV_GREEN" />
    <param name="own_ip" value="192.168.10.81" />
    <param name="udp_port" type="int" value="27002" />
  </node>

  <node pkg="vicon_bridge" type="vicon_bridge" name="vicon" output="screen">
    <param name="stream_mode" value="ClientPull" type="str" />
    <param name="datastream_hostport" value="192.168.10.1:801" type="str" />
    <param name="tf_ref_frame_id" value="/world" type="str" />
  </node>

  <node pkg="rxtools" type="rxplot" name="rxplot" args="/udp/vicon/QAV_GREEN/QAV_GREEN/transform/translation/x,/vicon/QAV_GREEN/QAV_GREEN/transform/translation/x" />

</launch>
  
